### PR TITLE
feat: Hide 'Hard' and 'Easy' buttons setting in the new reviewer

### PR DESCRIPTION
**Pull request by BrayanDSO**
It is a frequently requested feature, helps a big number of users that only use two buttons, and it is quite simple to implement.

Apparently, Damien don't want to implement it in Anki for some kind of reason, but it takes a lot of less time to implement it than to read the more than 100 comments in the corresponding page at the forums to still reach to the same conclusion.

About the PR, if any string related changes are required, please push them or do them in another PR, because I don't have time to bikeshedding and won't do them.

## How Has This Been Tested?

Emulator 35:

<details><summary>Screenshot</summary>

Screenshot_20240804_182727

</details>

## Checklist
_Please, go through these checks before submitting the PR._

- [X] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [X] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by lukstbit — APPROVED** [review]
We probably shouldn't diverge with desktop but I see no harm in implementing this so LGTM.

**Comment by voczi** [comment]
> We probably shouldn't diverge with desktop

Pass/fail addon on desktop *should* be the standard for people who aren't familiar with the meaning of the 4 buttons, i.e. most of the non-expert users.

With this setting on I would prefer labeling the buttons as Pass/Fail instead of Good/Again, though.

**Comment by BrayanDSO** [comment]
In my opinion, we shouldn't diverge in things that require an investment from the user and that can break their workflow, like tts or FEN, where the user had to add `<tts>` or `[FEN]` in their cards, and removing the features means breaking their cards.

But we also shouldn't be bottlenecked by every Anki's decision (or the lack of one). We should take it in account, but we shouldn't blindly follow it. 

I don't see any harm in this feature, only benefits. And we won't break anything if it needs to be replaced in the future.

About expectancies, if someone likes the feature and wants them in the desktop version, I think that is completely fair for them to request it there. 

---

I'm not sure exactly what the `Needs author reply` label is for, but echoing myself:

> About the PR, if any string related changes are required, please push them or do them in another PR, because I don't have time to bikeshedding and won't do them.

That includes renaming stuff to `Pass/Fail`, `Correct/Wrong` or whatever. Also, this is currently a developer option, so it can be put under an "addons" menu or whatever in the future.

**Comment by voczi** [comment]
How does a suggestion, aimed to improve the user experience and reduce cognitive load even further, fall under "bikeshedding"?

**Comment by BrayanDSO** [comment]
> How does a suggestion, aimed to improve the user experience and reduce cognitive load even further, fall under "bikeshedding"?

That's relative. Choosing words is bikeshedding to me. If someone else think that's important, I find it completely fine, but I don't want to spend time on it. As I said, please do a PR for that or push it here. If the PR can't be merged because of that, close it.

**Comment by voczi** [comment]
Ok.
Only reason why I didn't make a PR was because a discussion would ensue, for which I probably won't have time to engage in. In any case, a PR is here and hopefully it will be merged eventually.

**Comment by brishtibheja** [comment]
@voczi Fail/Wrong are negative words and shouldn't be used to replace the current Again. I don't think it increases cognitive load or anything like that. Plus, if this is eventually done in the desktop app we probably won't get a name change. Name change isn't much beneficial and all it does is make a whole lot of previously written tutorials/guides get outdated.

**Review by david-allison — APPROVED** [review]
TL;DR: I'd like to wait until Aug 16 for further comment [PR is obviously correct functionally, ecosystem-wise is a more difficult question], then merge as-is

I'm +1:

* Before FSRS raised the issue, two buttons has been popular for reducing decision fatigue while reviewing
   * I maintained a fork with Pass/Fail for a while 
* We should provide users with the ability to customize their learning experience, and we can't do this to the fullest extent as addons are not yet supported.

To be clear: I do **not** support making this a default unless it's done in-step with the rest of the ecosystem

----

>  this is currently a developer option, so it can be put under an "addons" menu or whatever in the future.

This works for me

I'd block the general feature release on moving it to an addons menu, to shield ourselves/upstream from the impact of divergence (and string changes/instability). I'm happy to work on this.

----

@mikehardy is on holiday and I'd want to be sure this PR is seen and give more than a quick glance before merging due to the long-term impact on project direction

**Review by mikehardy — APPROVED** [review]
I like this as a non-default option, I agree with the scope of the divergence being ... "non-fatal" if that's an acceptable term by the reasoning @BrayanDSO used with regard to us removing it wouldn't just break someone's template, it's just a visual element hide really.

Not accepting wording changes as a stated condition on the PR is pretty assertive Brayan ;-) at the same time allowing people to just push changes makes that assertive-but-okay esp when judged with the mountain of reviewer work you're doing. Word choices are the smallest detail in that context so works for me :-)

**Comment by mikehardy** [comment]
(just closing/reopening as I changed the set of github actions that run / are required, this re-triggers CI with the new set)

**Comment by mikehardy** [comment]
Strings are in sync, there are 3 green checks here for approval, I'm going to merge this and sync (I need to qualify i18n sync still works after some changes there so this is useful to me) then try for a fresh alpha
